packages/conventional-changelog-cli/test/fixtures/promise-config.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+var Q = require('q');
```

Comment:
Is `q` needed? Could we just use native promises?

Reply:
I guess we could use native promises. I only used `q` to match the rest of the code base and I'm also not familiar with native promises availability in given versions of `node` so I didn't want to muddle any further. This is only like my third pull request and the others were simple readme changes. lol...

Reply:
> This is only like my third pull request and the others were simple readme changes.

Ah, well thank you for taking the next step with this change.

> I only used q to match the rest of the code base 

That's fine. Since `conventional-changelog` only supports Node 4 and higher, so using native promises is perfectly fine, and my preference when possible (avoids third-party packages for native features where possible, lowering the overall overhead of installing `conventional-changelog`).

Though I'm not sure how much the other core maintainers of `conventional-changelog` prefer maintaining consistency within files.